src/Resources/translations/messages.en.yml

```diff
@@ -6,3 +6,4 @@ bitbag_sylius_product_attribute_group_plugin:
         edit_group: Edit attribute group
         attributes: Attributes
         no_attributes: No attributes asigned
+        attributes_group: Add attribute group
```

Comment:
you can add also polish translations, It will be needed on our project

Reply:
added